config/routes.rb

```diff
@@ -1,8 +1,11 @@
 Rails.application.routes.draw do
-  get 'home/login'
-  root 'home#index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+  get 'users', to: 'users#index', as: 'users'
+  get 'users/:id', to: 'users#show', as: 'user'
```

Comment:
- [OPTIONAL] Hey! You can simplify your routes code like this:

```ruby
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```

By using the `resources` method with the appropriate options, you can achieve the same routing behavior more concisely. This helps keep your codebase clean and easy to manage. Keep up the good work! 🚀👨‍💻